doc/source/webapp_ui/dashboard/navigation/delete_dispatches.rst

```diff
@@ -0,0 +1,43 @@
+======================
+Deleting Dispatches
+======================
+
+Select checkboxes on the left of dispatch lines to delete records of those dispatches.
+
+.. warning:: Deleting a dispatch removes the dispatch and its results permanently from the database. Deletion cannot be undone.
```

Comment:
quick comment, deleting from UI is a soft delete, it only changes the DB to not show in UI and not delete it in DB. Note this behaviour might change in future.

Reply:
Is this just a display flag in the DB then? Is there a way to restore deleted dispatches to the UI?

Reply:
Yes correct. Not yet !